examples/jenkins-image-sample.groovy

```diff
@@ -0,0 +1,119 @@
+/**
+ * This script does nothing in particular (in fact, it deletes its work),
+ * but is meant to show actual usage of most of the API.
+ */
+
+openshift.withCluster() {
+
+    def saSelector1 = openshift.selector( "serviceaccount" )
+    saSelector1.describe()
+    def saSelector2 = openshift.selector( "sa/builder" )
+    def saSelector3 = openshift.selector( "sa", "builder" )
+    def saSelector4 = openshift.selector( "dc", [ "template" : "mytemplate" ] )
+    
+    def templateSelector = openshift.selector( "template", "mongodb-ephemeral")
+        
+    templateSelector.delete('--ignore-not-found')
+    openshift.delete('all,secret', '-l', 'template=mongodb-ephemeral-template', '--ignore-not-found')
```

Comment:
Unfortunately it does not work.  This does not seem to anything over or above what the templateSelector.delete did.

Switching back to what I had for now.  If time permits, I'll circle back to this and see if I can make plugin changes to have this work.